101-functions-managed-identity/azuredeploy.json

```diff
@@ -0,0 +1,286 @@
+                "E. Australia Standard Time",
+                "AUS Eastern Standard Time",
+                "West Pacific Standard Time",
+                "Tasmania Standard Time",
+                "Vladivostok Standard Time",
+                "Lord Howe Standard Time",
+                "Bougainville Standard Time",
+                "Russia Time Zone 10",
+                "Magadan Standard Time",
+                "Norfolk Standard Time",
+                "Sakhalin Standard Time",
+                "Central Pacific Standard Time",
+                "Russia Time Zone 11",
+                "New Zealand Standard Time",
+                "UTC+12",
+                "Fiji Standard Time",
+                "Kamchatka Standard Time",
+                "Chatham Islands Standard Time",
+                "UTC+13",
+                "Tonga Standard Time",
+                "Samoa Standard Time",
+                "Line Islands Standard Time"
+            ],            
+            "metadata": {
+                "description": "Pick the timezone to use for the function"
+              }
+        }
+    },
+    "variables": {
+        "hostingPlanName": "[parameters('functionAppName')]",
+        "location": "[resourceGroup().location]",
+        "storageAccountName": "[concat('storage', uniquestring(resourceGroup().id))]"
+    },
+    "resources": [
+        {
+            "name": "[parameters('functionAppName')]",
+            "type": "Microsoft.Web/sites",            
+            "dependsOn": [
+                "[concat('Microsoft.Web/serverfarms/', variables('hostingPlanName'))]",
+                "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]",
+                "[resourceId('microsoft.insights/components/', parameters('functionAppName'))]"
+            ],
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "siteConfig": {
+                    "appSettings": [
+                        {
+                            "name": "FUNCTIONS_WORKER_RUNTIME",
+                            "value": "[parameters('runtimeStack')]"
+                        },
+                        {
+                            "name": "AzureWebJobsStorage",
+                            "value": "[concat('DefaultEndpointsProtocol=https;AccountName=',variables('storageAccountName'),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2015-05-01-preview').key1)]"
+                        },
+                        {
+                            "name": "FUNCTIONS_EXTENSION_VERSION",
+                            "value": "~2"
+                        },
+                        {
+                            "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+                            "value": "[reference(resourceId('microsoft.insights/components/', parameters('functionAppName')), '2015-05-01').InstrumentationKey]"
```

Comment:
update apiVersion

Reply:
This seems to be the latest supported version

Reply:
see other comment on how to get the latest